app/_assets/stylesheets/page.less

```diff
@@ -410,50 +405,63 @@
     font-style: italic;
   }
 
+    /*
+  Generic Styling for Desktop
+  */
   table {
     width: 100%;
-    border: 1px solid @gray;
-
-    th, td {
-      .font-source-sans();
-      font-size: 14px;
-      vertical-align: top;
-    }
+    border-collapse: collapse;
+    font-size: 14px;
+  }
+  code {
+    font-size: 14px;
+  }
+  /* Zebra striping */
+  tr:nth-of-type(odd) {
+    background: @gray-light;
+  }
+  th {
+    font-weight: bold;
+  }
+  td, th {
+    padding: 6px;
+    border: 1px solid #ccc;
+    text-align: left;
+  }
 
-    th {
-      text-transform: lowercase;
-      .font-source-sans-sc();
-      font-size: 14px;
-      text-align: left !important;
-      background: @gray-light;
-      border-bottom: 1px solid @gray;
-      color: lighten(@gray-dark, 30%);
-      padding: 10px;
+  @media
+  only screen and (max-width: 1099px)
```

Comment:
Is there a reason to break the line here?

Reply:
ah good catch, I had it when I had more settings so it would stay at 80 chars, but removed the additional settings